src/patterns/dosdp-patterns/location.yaml

```diff
@@ -7,14 +7,15 @@ description: >-
     
 classes:
     disease: MONDO:0000001
+    anatomical entity: UBERON:0001062
+    cell: CL:0000000
 
-    owl_thing: owl:Thing
 relations:
     disease has location: RO:0004026
 
 vars:
     disease: "'disease'"
-    location: owl_thing
+    location: "'anatomical entity'" or "'cell'"
```

Comment:
Should be `location: "'anatomical entity' or 'cell'"`

Reply:
thanks @dosumis!